templates/cloudformation/apps/aem/full-set/monitoring.yaml

```diff
@@ -15,37 +15,429 @@ Resources:
+                "region": "ap-southeast-2",
+                "title": "ELB RequestCount Metrics - Graph",
+                "period": 300
+              }
+            },
+            {
+              "type": "metric",
+              "x": 0,
+              "y": 33,
+              "width": 12,
+              "height": 6,
+              "properties": {
+                "metrics": [
+                  [ "System/Linux", "DiskSpaceUtilization", "MountPath", "/mnt/ebs1", "InstanceId", "${authorPrimaryInstance}", "Filesystem", "/dev/xvdb" ],
+                  [ "...", "${authorStandbyInstance}", "Filesystem", "/dev/xvdb" ],
+                  [ "System/Linux", "DiskSpaceUtilization", "MountPath", "/mnt/ebs1", "AutoScalingGroupName", "${authorDispatcherAutoScalingGroupName}", "Filesystem", "/dev/xvdb" ],
+                  [ "...", "${publishAutoScalingGroupName}", "Filesystem", "/dev/xvdb" ],
+                  [ "...", "${publishDispatcherAutoScalingGroupName}", "Filesystem", "/dev/xvdb" ],
+                  [ "...", "${orchestratorAutoScalingGroupName}", "Filesystem", "/dev/xvdb" ],
+                  [ "...", "${choasMonkeyAutoScalingGroupName}", "Filesystem", "/dev/xvdb" ]
+                ],
+                "view": "timeSeries",
+                "stacked": false,
+                "region": "ap-southeast-2",
+                "title": "DiskSpaceUtilization on Data Volume /mnt/ebs1"
+              }
+            },
+            {
+              "type": "metric",
+              "x": 12,
+              "y": 33,
+              "width": 12,
+              "height": 6,
+              "properties": {
+                "metrics": [
+                  [ "System/Linux", "DiskSpaceUtilization", "MountPath", "/", "InstanceId", "${authorPrimaryInstance}", "Filesystem", "/dev/xvda2" ],
+                  [ "...", "${authorStandbyInstance}", "Filesystem", "/dev/xvda2" ],
+                  [ "System/Linux", "DiskSpaceUtilization", "MountPath", "/", "AutoScalingGroupName", "${authorDispatcherAutoScalingGroupName}", "Filesystem", "/dev/xvda2" ],
+                  [ "...", "${publishAutoScalingGroupName}", "Filesystem", "/dev/xvda2" ],
+                  [ "...", "${publishDispatcherAutoScalingGroupName}", "Filesystem", "/dev/xvda2" ],
+                  [ "...", "${orchestratorAutoScalingGroupName}", "Filesystem", "/dev/xvda2" ],
+                  [ "...", "${choasMonkeyAutoScalingGroupName}", "Filesystem", "/dev/xvda2" ]
+                ],
+                "view": "timeSeries",
+                "stacked": false,
+                "region": "ap-southeast-2",
+                "title": "DiskSpaceUtilization on root volume"
+              }
+            },
+            {
+              "type": "metric",
+              "x": 0,
+              "y": 39,
+              "width": 12,
+              "height": 6,
+              "properties": {
+                "view": "timeSeries",
+                "stacked": false,
+                "region": "ap-southeast-2",
```

Comment:
Region should come from parameter/variable.

Reply:
Replaced region value with variable ${awsRegion}